deploy/balena-k3s/gpu/entry.sh

```diff
@@ -0,0 +1,18 @@
+# Remove Nouveau modules
+sleep 6
+echo 0 > /sys/class/vtconsole/vtcon1/bind
+sleep 2
+rmmod nouveau
+sleep 2
+
+# see https://forums.balena.io/t/blacklist-drivers-in-host-os/163437/25
+
+# Insert Nvidia modules
+insmod /nvidia/driver/nvidia.ko
+insmod /nvidia/driver/nvidia-modeset.ko
+insmod /nvidia/driver/nvidia-uvm.ko
+
+/usr/bin/nvidia-smi
+nvidia-modprobe
+
+sleep infinity
```

Comment:
Why not just let this container exit and set the docker-compose definition to `restart: "no"`? I don't know that makes a huge difference, but it does express intent more clearly.

It also means we could use 
```
depends_on:
  gpu:
    condition: service_completed_successfully
``` 
to have other containers wait on this getting set up, but I don't know if we need that here.

Reply:
It turns out that Balena doesn't allow this even though its valid in docker-compose